src/components/Select/Select.stories.tsx

```diff
@@ -10,16 +10,14 @@ const selectData = [
   { label: "Dogecoin", value: "doge" },
 ]
 
-type SelectType = typeof SelectComponent
-
-const meta: Meta<SelectType> = {
+const meta = {
   title: "Atoms / Form / Select",
   component: SelectComponent,
-}
+} satisfies Meta<typeof SelectComponent>
 
 export default meta
 
-type Story = StoryObj<SelectType>
+type Story = StoryObj<typeof meta>
 
 export const Select: Story = {
   args: {
```

Comment:
@TylerAPfledderer Getting some merge conflicts on this file (`Select.stories.tsx`)... I think we probably want to overwrite with what we have here, but I'm getting a type error with `minW` in the args when I try to do that, since we seem to be attempting to spread that onto ReactSelect (not a Chakra-UI component).

I think we need to remove this `minW` arg, but wanted to check with you.

Reply:
This version of the select stories file is outdated and that arg no longer exists. Fixed the conflict to ensure the file currently in prod stays intact. (It's now removed from the diffs in this PR, btw)